rllib/policy/tf_policy.py

```diff
@@ -66,6 +66,18 @@ class TFPolicy(Policy):
         SampleBatch({"action": ..., "advantages": ..., ...})
     """
 
+    # In order to create tf_policies from checkpoints, this class needs to separate
+    # variables into their own scopes. Normally, we would do this in the modul
+    # catalog, but since Policy.from_state() can be called anywhere, we need to
+    # keep track of it here to not break the from_state API.
+    tf_var_creation_scope_counter = 0
+
+    @staticmethod
+    def next_tf_var_scope_name():
+        # Tracks multiple instances that are spawned from this policy via .from_state()
+        TFPolicy.tf_var_creation_scope_counter += 1
+        return "tf_policy_var_scope_" + str(TFPolicy.tf_var_creation_scope_counter)
```

Comment:
nit ``f"tf_policy_var_scope_{TFPolicy.tf_var_creation_scope_counter}"``
also, just to double check, should we use a shorter scope name? this name becomes part of the variable name right? maybe ``f"var_scope_{TFPolicy.tf_var_creation_scope_counter}"``